src/pudl/transform/eia861.py

```diff
@@ -263,10 +1064,14 @@ def transform(raw_dfs, tables=pc.pudl_tables["eia861"]):
                     "Not transforming EIA 861.")
         return tfr_dfs
     # for each of the tables, run the respective transform funtction
-    for table in eia861_transform_functions:
-        if table in tables:
-            logger.info(f"Transforming raw EIA 861 DataFrames for {table} "
-                        f"concatenated across all years.")
-            eia861_transform_functions[table](raw_dfs, tfr_dfs)
-
+    for table in eia861_tables:
+        logger.info(f"Transforming raw EIA 861 DataFrames for {table} "
+                    f"concatenated across all years.")
+        # ADD some sort of customer message here / check across multiple sources
+        assert table in tfr_funcs.keys()
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/zaneselvans/pudl/pullRequest?prid=5664147)